### DISCUSSION TITLE
How is Capacity calculated?

### DISCUSSION THREAD

**Discussion by wass-cloud** [pybamm-team/PyBaMM#3199] [Q&A]
I understand that certain parameters affect the capacity, but as a beginner coder PyBaMM's dense code makes it hard for me to find stuff. Is there some sort of formula that PyBaMM uses to calculate the capacity of a battery from certain parameters (e.g electrode thickness, volume fraction, etc.)? 

So far most of the models I've run that have reliable physical parameters do not behave as expected in PyBaMM.

**Comment by valentinsulzer** [comment]
This might help https://docs.pybamm.org/en/latest/source/examples/notebooks/models/electrode-state-of-health.html

**Reply by wass-cloud** [reply]
It sort of did, however I'm still unsure as to how capacity is affected by the size of the cell/electrodes. What is meant by x0, x100, y0, and y100?

**Reply by brosaplanella** [reply]
The technical details are explained in detail in the article referenced in the notebook: 
> Mohtat, P., Lee, S., Siegel, J. B., & Stefanopoulou, A. G. (2019). Towards better estimability of electrode-specific state of health: Decoding the cell expansion. Journal of Power Sources, 427, 101-111.
